forms-flow-theme/src/global.css

```diff
@@ -622,7 +622,7 @@ div.upload input[type="file"] {
 }
 
 .admin-container > .header-container {
-  padding-top: 5rem !important;
+  padding-top: 10px !important;
```

Comment:
We should stick to standardise using px vs REm across and at relevant places: Use px for small, fixed-size elements like borders or shadows.
Use em for typography and other scalable elements that need to change size relative to their parent element.
Use rem for scalable typography and responsive layouts that need to change size relative to the root element.